.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,122 @@
+  DREMIO_PORT: ${{ secrets.DREMIO_PORT }}
+  DREMIO_SOFTWARE_USER: ${{ secrets.DREMIO_SOFTWARE_USER }}
+  DREMIO_PW: ${{ secrets.DREMIO_SOFTWARE_PASSWORD }}
+  DREMIO_SOFTWARE_PAT: ${{ secrets.DREMIO_SOFTWARE_PAT }}
+  DREMIO_CLOUD_PAT: ${{ secrets.DREMIO_CLOUD_PAT }}
+  DREMIO_CLOUD_USER: ${{ secrets.DREMIO_CLOUD_USER }}
+  DREMIO_CLOUD_PROJECT: ${{ secrets.DREMIO_CLOUD_PROJECT }}
+  DREMIO_SOURCE: ${{ secrets.DREMIO_SOURCE}}
+  DREMIO_SPACE: ${{ secrets.DREMIO_SPACE}}
+  DBT_TEST_USER_1: ${{ secrets.DBT_TEST_USER_1}}
+  DBT_TEST_USER_2: ${{ secrets.DBT_TEST_USER_2}}
+  DBT_TEST_USER_3: ${{ secrets.DBT_TEST_USER_3}}
+
+jobs:
+  run-functional-tests_cloud:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout the project
+        uses: actions/checkout@v2
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.10.0
+      - name: Install requirements
+        run: pip install -r dev_requirements.txt
+      - name: Set env variables
+        run: |
+          echo "DREMIO_CLOUD_HOST=$DREMIO_CLOUD_HOST" >> $GITHUB_ENV
+          echo "DREMIO_SOFTWARE_HOST=$DREMIO_SOFTWARE_HOST" >> $GITHUB_ENV
+          echo "DREMIO_CLOUD_USERNAME=$DREMIO_CLOUD_USER" >> $GITHUB_ENV
+          echo "DREMIO_SOFTWARE_USERNAME=$DREMIO_SOFTWARE_USER" >> $GITHUB_ENV
+          echo "DREMIO_SOFTWARE_PASSWORD=$DREMIO_SOFTWARE_PASSWORD" >> $GITHUB_ENV
+          echo "DREMIO_PAT=$DREMIO_CLOUD_PAT" >> $GITHUB_ENV
+          echo "DREMIO_CLOUD_PROJECT_ID=$DREMIO_CLOUD_PROJECT" >> $GITHUB_ENV
+          echo "DREMIO_DATALAKE=$DREMIO_SOURCE" >> $GITHUB_ENV
+          echo "DREMIO_DATABASE=$DREMIO_SPACE" >> $GITHUB_ENV
+          echo "DBT_TEST_USER_1=$DBT_TEST_USER_1" >> $GITHUB_ENV
+          echo "DBT_TEST_USER_2=$DBT_TEST_USER_1" >> $GITHUB_ENV
+          echo "DBT_TEST_USER_3=$DBT_TEST_USER_1" >> $GITHUB_ENV
+      - name: Run Grants Tests
+        run: pytest ./tests/functional/adapter/grants --profile dremio_cloud -vv
+      - name: Run Basic Tests
+        run: pytest ./tests/functional/adapter/basic --profile dremio_cloud -vv
+
+  run-unit-tests:
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout the project
+          uses: actions/checkout@v2
+        - name: Set up Python 3.10
+          uses: actions/setup-python@v2
+          with:
+            python-version: 3.10.0
+        - name: Install requirements
+          run: pip install -r dev_requirements.txt
+        - name: Run unit Tests
+          run: pytest ./tests/unit/
+
+  run-softwareUP-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the project
+        uses: actions/checkout@v2
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.10.0
+      - name: Install requirements
+        run: pip install -r dev_requirements.txt
+      - name: Install current dbt-dremio
+        run: pip install -e .
```

Comment:
Does pip install -e work here?!

Reply:
Yup, seems to work fine in github actions.